packages/wrangler/src/pipelines/index.ts

```diff
@@ -237,13 +244,27 @@ export function pipelines(pipelineYargs: CommonYargsArgv) {
 					throw new FatalError("Requires a r2 secret access key");
 				}
 
-				if (args.authentication) {
-					pipelineConfig.source = [
-						{
-							type: "binding",
-							format: "json",
+				// add binding source (default to add)
+				if (args.binding === undefined || args.binding) {
+					pipelineConfig.source.push({
+						type: "binding",
+						format: "json",
+					} satisfies BindingSource);
+				}
+
+				// add http source (possibly authenticated), default to add
+				if (args.http === undefined || args.http) {
+					pipelineConfig.source.push({
+						type: "http",
+						format: "json",
+						config: {
+							authentication:
+								args.authentication !== undefined && args.authentication,
 						},
-					];
+					} satisfies HttpSource);
+				}
+				if (pipelineConfig.source.length < 1) {
+					throw new FatalError("At leat one source should be specified");
```

Comment:
is this a `UserError` or a `FatalError`. The difference is that the latter will be tracked in Sentry, whereas the former won't

Reply:
I also wonder if we could make this err message more actionable on the user side. So for example, can we point out how they should specify a source? either by providing an example or linking to some docs?